sickbeard/server/web/config/search.py

```diff
@@ -69,7 +69,7 @@ def saveSearch(self, use_nzbs=None, use_torrents=None, nzb_dir=None, sab_usernam
         sickbeard.BACKLOG_DAYS = try_int(backlog_days, 7)
 
         sickbeard.CACHE_TRIMMING = config.checkbox_to_value(cache_trimming)
-        sickbeard.MAX_CACHE_AGE = float(max_cache_age)
+        sickbeard.MAX_CACHE_AGE = try_int(max_cache_age)
```

Comment:
hmm, then we should change those everywhere. As everywhere try_int is used. Also for ex. BACKLOG_DAYS and USENET_RETENTION.